Validation/RecoTau/plugins/DQMFileLoader.h

```diff
@@ -21,7 +21,7 @@
 #include <vector>
 #include <string>
 
-class TauDQMFileLoader : public edm::EDAnalyzer {
+class TauDQMFileLoader : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
   typedef dqm::legacy::DQMStore DQMStore;
   typedef dqm::legacy::MonitorElement MonitorElement;
```

Comment:
Is the direct use of `DQMStore` safe in non-legacy modules?

Reply:
I believe it is not. I leave the author of the code to make use of DQM codes``

Reply:
Then, to me, it would make more sense to leave these modules as legacy and migrate them properly (or remove) later rather than enabling unsafe code.

Reply:
Done that